jupyterlab/labbox_ephys_widgets_jp/labbox_ephys_widgets_jp/create_recording_view.py

```diff
@@ -13,16 +13,25 @@
 
 from ._frontend import module_name, module_version
```

Comment:
Avoid code duplication by using the WorkerSession from GUI for handling hither calls in jupyter widgets.